tournaments/index.js

```diff
@@ -512,6 +512,11 @@ class Tournament {
 		this.inProgressMatches.forEach((match, playerFrom) => {
 			if (match && match.to === player) matchTo = playerFrom;
 		});
+		if (this.room.isOfficial && this.generator.users.size >= 2) {
+			let matchRoom = matchFrom === null ? this.inProgressMatches.get(matchTo).room : matchFrom.room;
```

Comment:
``null`` is a falsey value, so if it's null, then it will use ``matchFrom.room``